From 86ec35c1c09f3ef98cab669915a9aec7a7ddb275 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sun, 8 Aug 2021 10:19:01 +0200
Subject: [PATCH 135/N] sysconfig: fix platlib/purelib paths

make them match what we use in distutils. pip is moving from distutils
to sysconfig and this is breaking things.
---
 Lib/sysconfig.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Lib/sysconfig.py b/Lib/sysconfig.py
index 854b382..0729dc9 100644
--- a/Lib/sysconfig.py
+++ b/Lib/sysconfig.py
@@ -50,8 +50,8 @@ _INSTALL_SCHEMES = {
     'nt': {
         'stdlib': '{installed_base}/lib/python{py_version_short}',
         'platstdlib': '{base}/lib/python{py_version_short}',
-        'purelib': '{base}/lib/python{py_version_short}',
-        'platlib': '{base}/lib/python{py_version_short}',
+        'purelib': '{base}/lib/python{py_version_short}/site-packages',
+        'platlib': '{base}/lib/python{py_version_short}/site-packages',
         'include': '{installed_base}/include/python{py_version_short}',
         'platinclude': '{installed_base}/include/python{py_version_short}',
         'scripts': '{base}/bin',
-- 
2.32.0

